### PR TITLE
[hacktoberfest] solution for game randow number [Java]

### LINKED ISSUE #257: Guessing Game in C language
I want to add a small code in the **C language** of **Guessing Game**.

- It is basically a guessing game that allows users to enter any random number within a specific range.

- It also provides some chances to users so that if in case their first guess is wrong they may try one more time.

If you like me to add this code to your repo. Kindly allow me to raise PR.
Thankyou!

### PR DESCRIPTION
**Pull request by jackson-roberio**
# [GAME] What is Number? :video_game:

I made a program in Java, where the player must guess the number generated randomly by the program, this number is an integer and can be between the value 1 to the value 20.

The player will have 5 attempts to set the randomly generated number, if after the 5 attempts the player does not get it right, then the game informs the hidden number and the game is finished.

Fixes #257

## Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# Checklist:

-   [ ] My code follows the style guidelines of this project
-   [X] I have performed a self-review of my own code
-   [X] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Java/WhatIsNumberGame.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package whatnumber;
```
Comment (kaustubhgupta) [inline Java/WhatIsNumberGame.java]: Just update the readme of the Java folder with your entry, everything else LGTM :-)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Java/WhatIsNumberGame.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
+package br.com.jacksonroberio;
```
Comment (YashSaxena9) [inline Java/WhatIsNumberGame.java]: Remove package name, this file is independent of package :)

**Review by YashSaxena9 — CHANGES_REQUESTED** [review]
Your code looks good, please make the required changes :)

**Comment by kaustubhgupta** [comment]
@jackson-roberio update java folder readme also

File: Java/WhatIsNumberGame.java (hunk 2 above)
Comment (jackson-roberio) [reply Java/WhatIsNumberGame.java]: ok, done

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
Great thanks for contribution :-)

**Comment by jackson-roberio** [comment]
> Great thanks for contribution :-)

Thanks for the opportunity

**Pull request merged**